FlowCrypt/Extensions/CombineExtensions.swift

```diff
@@ -25,3 +25,14 @@ extension Publisher {
             })
     }
 }
+
+extension Publishers.SubscribeOn {
+    // swiftlint:disable line_length
+    func myFlatMap<P>(_ transform: @escaping (Self.Output) -> P) -> Publishers.FlatMap<P, Publishers.SetFailureType<Self, P.Failure>> where P: Publisher {
```

Comment:
Publishers.SubscribeOn flatMap operator available only in iOS 14 and higher so I have to create this one

Reply:
It was another problem and with SO I found how to handle it correctly.